src/frontend/magnify/packages/components/src/components/rooms/RoomConfig/RoomConfig.tsx

```diff
@@ -122,6 +126,9 @@ export const RoomConfig = ({ room }: RoomConfigProps) => {
           return newRooms;
         });
       },
+      onError: (error) => {
+        modals.handleNotLoggedRequest(error);
```

Comment:
I would maybe have used an abstraction between errors and modals. Because: what if some day it isn't modals that are used for errors ? But Toast, etc 

Like this: 

```
// errors/index.ts

function useErrors() {
	const modals = useModals();
	return {
		onError: (error) => {
			modals.onError(error);
			// Do other tasks such as report on Sentry, etc .. 
		}
	}
}
```